task5/servlets/CheckServlet.java

```diff
@@ -0,0 +1,26 @@
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.io.PrintWriter;
+
+@WebServlet("/check")
+public class CheckServlet extends HttpServlet {
+    protected void doPost(HttpServletRequest request, HttpServletResponse response)
```

Comment:
Always use `@Override`  annotation with the methods that - well - override methods from a superclass)